libs/dh/shared/data-access-api/src/lib/generated/v1/model/metering-point-simple-cim-dto.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * DataHub BFF
+ * Backend-for-frontend for DataHub
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+import { ConnectionState } from './connection-state';
+
+
+export interface MeteringPointSimpleCimDto { 
+    meteringPointId?: string;
+    gsrnNumber?: string;
+    connectionState?: ConnectionState;
+    effectiveDate?: string;
+}
```

Comment:
issue: In order to implement the child metering point overview table, we need more data which is not currently part of the interface. Looking at the sketch, I can identify the following data:

![image](https://user-images.githubusercontent.com/1096332/145993413-63770566-bfc5-43f4-8d22-f9f770707d36.png)

1. Metering Point Type
2. GRSN Number
3. Effective Date (Start Date?)
4. End Date (?)
5. Connection State

The interface contains data about 2 (`gsrnNumber`), 3 (`effectiveDate`) and 5 (`connectionState`) but it misses 1 (`meteringPointType`) and 4 (new property).

Is the metering point domain able to supply the missing data?

Reply:
properties added